src/ir/printer.rs

```diff
@@ -163,16 +163,16 @@ impl<'a> Printer<'a> {
 
     fn access(&self, a: &ir::Access) -> String {
         let &ir::Access { port, start, end } = a;
-        if a.is_port(self.ctx) {
-            self.ctx.display(port)
-        } else {
-            format!(
-                "{}[{}..{})",
-                self.ctx.display(port),
-                self.expr(start),
-                self.expr(end)
-            )
-        }
+        // if a.is_port(self.ctx) {
+        //     self.ctx.display(port)
+        // } else {
+        format!(
+            "{}[{}..{})",
+            self.ctx.display(port),
+            self.expr(start),
+            self.expr(end)
+        )
+        //}
```

Comment:
Commented out code here. If this is debugging related, let's move it under `log::debug_enabled!`